lib/network/http2_network.go

```diff
@@ -61,15 +69,22 @@ func NewHTTP2Network(config HTTP2NetworkConfig) (h2n *HTTP2Network) {
 		},
 	)
 
+	baseRouter := mux.NewRouter()
+
 	h2n = &HTTP2Network{
 		server:         server,
+		router:         baseRouter,
 		tlsCertFile:    config.TLSCertFile,
 		tlsKeyFile:     config.TLSKeyFile,
 		receiveChannel: make(chan Message),
 	}
+	h2n.handlers = map[string]func(http.ResponseWriter, *http.Request){}
+	h2n.routers = map[string]*mux.Router{
+		"node": baseRouter.PathPrefix("/node").Subrouter(),
+		"api":  baseRouter.PathPrefix("/api").Subrouter(),
```

Comment:
We can use `UrlPrefixXX` in this place even in map key.

Reply:
Yeap. I changed it to named const.